2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,24 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends.forEach(friend => {
+    if (friend.age >= 35) {
+        thirtyFiveOrOlder.push(friend);
+    }
+});
+
```

Comment:
Great use of forEach method and if statement. I have used the filter method to get the same solution. Both work great.